fake/src/faker/impls/internet.rs

```diff
@@ -136,6 +136,21 @@ impl<L: Data> Dummy<IP<L>> for IpAddr {
     }
 }
 
+impl<L: Data> Dummy<MACAddress<L>> for String {
+    fn dummy_with_rng<R: Rng + ?Sized>(_: &MACAddress<L>, rng: &mut R) -> Self {
+        let u = Uniform::new_inclusive(u8::min_value(), u8::max_value());
+        format!(
+            "{:X}:{:X}:{:X}:{:X}:{:X}:{:X}",
```

Comment:
The format should be {:02x} instead? so that when the value is 0-9, it will pad with zero, i.e. 1 become 01

Reply:
Oh I see, I've updated the format